products/unity.md

```diff
@@ -15,6 +15,13 @@ changelogTemplate: "https://unity.com/releases/editor/whats-new/{{'__LATEST__'|s
 
 releases:
 -   releaseCycle: "6"
+    releaseDate: 2024-04-29
+    eol: false
+    latest: "6000.1.0f1"
+    latestReleaseDate: 2025-04-23
+
+-   releaseCycle: "6"
```

Comment:
```suggestion
-   releaseCycle: "6.0"
```

Reply:
Not even unity itself calls it 6.1 , they are under `Unity 6` name . So 6.1 or 6.0 naming is -1 for my 2 cents